packages/admin-ui-docs/content/filter/code.mdx

```diff
@@ -0,0 +1,305 @@
+import { Section } from '@site/src/components/Section'
+
+<Section hasExplanation>
+
+## Usage
+
+```jsx
+import {
+  experimental_FilterDisclosure as FilterDisclosure,
+  experimental_FilterPopover as FilterPopover,
+  experimental_FilterListbox as FilterListbox,
+  experimental_FilterOptionRadio as FilterOptionRadio,
+  experimental_FilterOptionCheckbox as FilterOptionCheckbox,
+  experimental_FilterFooter as FilterFooter,
+  experimental_useFilterMultipleState as useFilterMultipleState,
+  experimental_useFilterState as useFilterState,
+  experimental_useFilterGroupState as useFilterGroupState,
+  experimental_FilterGroup as FilterGroup,
+} from '@vtex/admin-ui'
+
+function Example() {
+  const filterState = useFilterState()
+  const multipleSelectState = useFilterMultipleState()
+
+  const filterGroupState = useFilterGroupState({
+    filterStates: [state2, state],
+  })
+
+  return(
+    <FilterGroup>
+      // basic filter
+      <FilterDisclosure state={filterState}>Example</FilterDisclosure>
+
+      <FilterPopover state={filterState}>
+        <FilterListbox>
+          <FilterOptionRadio id="#1" label="Full" />
+          <FilterOptionRadio id="#2" label="Empty" />
+          <FilterOptionRadio id="#3" label="Half full" />
+        </FilterListbox>
+        <FilterFooter />
+      </FilterPopover>
+
+      // multiple selection filter
+      <FilterDisclosure state={multipleSelectState}>Example</FilterDisclosure>
+
+      <FilterPopover state={multipleSelectState}>
+        <FilterListbox>
+          <FilterOptionCheckbox id="#1" label="Full" />
+          <FilterOptionCheckbox id="#2" label="Empty" />
+          <FilterOptionCheckbox id="#3" label="Half full" />
+        </FilterListbox>
+        <FilterFooter />
+      </FilterPopover>
+    <FilterGroup />
+  )
+}
+```
+
+</Section>
+
+<Section hasExplanation>
+
+## State
+
+The state hooks `useFilterState` and `useFilterMultipleState` contain all business logic needed for
+controlling individual filters. The two hooks are different, but they share many props in common.
+
+The state hook `useFilterGroupState` handles the "clear all" action for the `FilterGroup` component.
+
+</Section>
+
+<Section>
+
+### useFilterState and useFilterMultipleState props
+
+| Name           | Type                                           | Description                                                                      | Required | Default  |
+| -------------- | ---------------------------------------------- | -------------------------------------------------------------------------------- | -------- | -------- |
+| fullList       | {id: string, label: string, value?: AnyObject} | Full option list that will be used for searching. Only use it if you need search.| 🚫       | -        |
+
+
+both filters return an object that extends the following type
+
+```ts
```

Comment:
What do you think of adding those types as Prop tables? You can name the prop table section title with the type name